tests/installation/partitioning_raid.pm

```diff
@@ -175,13 +175,12 @@ sub modify_uefi_boot_partition {
     assert_screen 'partitioning_raid-disk_vdd_with_partitions-selected';
     # fold the drive tree
     send_key 'left';
+    send_key 'left' if is_storage_ng;    # With storage-ng first press folds vdd
     assert_screen 'partitioning_raid-hard_disks-unfolded';
-    # select first partition of the first disk (usually vda1), bit of a short-cut
-    if (is_storage_ng) {
-        send_key 'left';
-        send_key 'right';
-    }
+    # select first partition of the first partition (usually vda1), bit of a short-cut
```

Comment:
"select first partition of the first partition" ?

Reply:
This is the old comment, let me improve it =)